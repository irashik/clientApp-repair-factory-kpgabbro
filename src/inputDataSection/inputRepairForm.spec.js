


import React from 'react';
import { shallow, render } from 'enzyme';

import InputRepairFrom from './inputRepairForm';



describe('<InputRepairFrom />', () => {

    test('snapShot correctly', () => {

        const component = shallow(<InputRepairFrom />);
        expect(component).toMatchSnapshot();

    });
});


