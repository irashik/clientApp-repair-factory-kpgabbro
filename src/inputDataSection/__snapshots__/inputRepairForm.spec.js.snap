// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InputRepairFrom /> snapShot correctly 1`] = `
<Modal
  animation={false}
  aria-labelledby="contained-modal-title-vcenter"
  autoFocus={true}
  backdrop="static"
  dialogAs={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "render": [Function],
    }
  }
  dialogClassName="modal-90w"
  enforceFocus={true}
  id="modalRepairForm"
  keyboard={true}
  restoreFocus={true}
  show={false}
  size="lg"
>
  <ModalHeader
    closeButton={false}
    closeLabel="Close"
  >
    <ModalTitle>
      Форма ввода данных о работах
    </ModalTitle>
    <button
      aria-label="Close"
      className="btn-close"
      id="btnCloseModal"
      onClick={[Function]}
      type="button"
    />
  </ModalHeader>
  <ModalBody
    className="show-grid"
    id="modalRepairForm-body"
  >
    <Container
      fluid={true}
      id="input-module"
    >
      <Row>
        <Col>
          <label>
            Время начала
          </label>
          <DatePickerDiv
            onSelectOptedData={[Function]}
            setDate=""
            type="dateStart"
          />
          <label>
            Время окончания
          </label>
          <DatePickerDiv
            onSelectOptedData={[Function]}
            setDate=""
            type="dateEnd"
          />
        </Col>
        <Col
          md="auto"
        >
          <FormControl
            id="inputEquipment"
            idunit=""
            onChange={[Function]}
            placeholder="Выберите оборудование"
            size="sm"
            type="text"
            value=""
          />
          <SearchList
            filter=""
            onSelectEquipment={[Function]}
          />
        </Col>
      </Row>
      <Col
        md={2}
      >
        <InputGroupButtonSmall
          name="repair"
          onHandleRepairCount={[Function]}
        />
      </Col>
      <Col>
        <FormInputRepair
          count={
            Array [
              1,
            ]
          }
          onHandleRepair={[Function]}
          onLoadRepair={Array []}
        />
      </Col>
      <Row>
        <Col>
          <FormInputMaterial
            count={
              Array [
                1,
              ]
            }
            onHandleMaterial={[Function]}
            onLoadMaterial={Array []}
          />
        </Col>
        <Col
          md={2}
        >
          <InputGroupButtonSmall
            name="material"
            onHandleMaterialCount={[Function]}
          />
        </Col>
      </Row>
      <Row>
        <AuthorView
          author=""
        />
      </Row>
    </Container>
  </ModalBody>
  <ModalFooter>
    <BtnView
      onLoadRecord=""
    />
  </ModalFooter>
</Modal>
`;
